src/js/plugins/ethereum/typedData.test.js

```diff
@@ -8,26 +8,28 @@ function messageToHex(string) {
 }
 
 describe('typedData', () => {
-    commonFixtures.tests
-        .filter(test => test.parameters.metamask_v4_compat)
-        .forEach(test => {
-            it('typedData to message_hash and domain_separator_hash', () => {
-                const transformed = typedData(
-                    test.parameters.data,
-                    test.parameters.metamask_v4_compat,
-                );
-                const { domain_separator_hash, message_hash } = transformed;
+    describe('should create message_hash and domain_separator_hash from typedData', () => {
+        commonFixtures.tests
+            .filter(test => test.parameters.metamask_v4_compat)
+            .forEach(test => {
+                it(test.name, () => {
```

Comment:
The main thing I changed here was to add another `describe()` call, and in the `it()` call use the fixture `test.name` as the test name. It's just to make the `yarn test:unit` results look prettier :)